Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeLibsAPI.cs

```diff
@@ -45,6 +62,13 @@ public static bool CompileComponent(IVBE ide, string projectName, string compone
                 return typeLibs.FindTypeLib(projectName).FindTypeInfo(componentName).CompileComponent();
             }
         }
+        public static bool CompileComponent(IVBProject project, string componentName)
```

Comment:
We could further overload this method taking a single parameter `IVBComponent` (inferring the project from its `VBProject` property.

Reply:
yeah, that might be handy.